src/tools-support/support.markdown

```diff
@@ -11,29 +11,14 @@ redirect_from: /docs-and-resources/support/
 * [Partners](/tools-support/requesting-partner-support.html)
 * [Developers (Developer Forum)](https://forum.developer.concur.com)
 * [Company Administrators](https://www.concur.com/en-us/contact)
-* **Individual Users** should request assistance from their company's internal help desk or their company's Concur administrator
+* **Individual Users** should request assistance from their company's internal help desk or their company's SAP Concur administrator
 
 ## Maintenance Schedule
 
-The Concur platform undergoes routine maintenance according to the following schedule. During these maintenance periods, the service may be unavailable. Unplanned maintenance is performed rarely, and limited to high priority fixes. During downtime, any request to Concur Connect will receive an HTTP 503 error code.
-
-#### Weekly maintenance window
-Every week, starting Friday at 9 PM PST and ending at 1 AM PST on Saturday
-
-#### Monthly maintenance schedule for 2017
-The service may be unavailable starting at 6 PM PST and ending at 11 PM PST on the following dates:
-
-* January 19
-* February 16
-* March 18
-* April 22
-* May 20
-* June 24
-* July 22
-* August 19
-* September 16
-* October 14
-* November 4
-* December 9
+The SAP Concur platform undergoes routine maintenance each weekend, and releases updates monthly. During these periods, the service may be unavailable. Unplanned maintenance is performed rarely, and limited to high priority fixes. During downtime, any request to the SAP Concur Platform will receive an HTTP 503 error code. To check the current status see: http://open.concur.com/, a message will be posted regarding planned maintenance.
```

Comment:
Let's say "a message will be posting during planned maintenance"

Reply:
Done